cli.js

```diff
@@ -51,6 +51,13 @@ var argv = require('yargs')
     default: defaults.silent,
     global: true
   })
+  .option('tag-prefix', {
+    alias: 't',
+    describe: 'Set a custom prefix for the git tag to be created',
+    type: 'boolean',
```

Comment:
If you switch the type to `string` rather than `boolean` I believe the tests should start passing.